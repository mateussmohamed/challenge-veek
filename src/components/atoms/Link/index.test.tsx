import React from 'react'

import { Link } from './index'
import withThemeProvider from '../../theme/withThemeProvider'

describe('[<Link />]', () => {
  const wrap = props => withThemeProvider(() => <Link {...props} />)

  test('matches to snapshot', () => {
    const wrapper = wrap({ children: 'I am a Link' })

    expect(wrapper).toMatchSnapshot()
  })

  describe('when pass props', () => {
    test('when pass color prop', () => {
      const wrapper = wrap({
        children: 'I am a Link with color',
        color: '#283583',
      })

      expect(wrapper).toMatchSnapshot()
    })
  })
})
