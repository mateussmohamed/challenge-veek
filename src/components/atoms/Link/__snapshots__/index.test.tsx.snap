// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[<Link />] matches to snapshot 1`] = `[Function]`;

exports[`[<Link />] when pass props when pass color prop 1`] = `[Function]`;
